sunpy/lightcurve/lightcurve.py

```diff
@@ -56,7 +56,7 @@ class LightCurve(object):
 
     References
     ----------
-    * `Pandas Documentation <http://pandas.pydata.org/pandas-docs/dev/dsintro.html>`_
+    * `Pandas Documentation <http://pandas-docs.github.io/pandas-docs-travis/>`_
```

Comment:
This one can be https, they are not changing the url. However, I would prefer to point to their stable documentation, not their development one: `https://pandas.pydata.org/pandas-docs/stable/`